ftp/1.5/modules/ROOT/pages/ftp-read.adoc

```diff
@@ -43,6 +43,9 @@ include::partial$common-read-operation.adoc[]
 // == File Locking SHARED BY FTP AND SFTP only
 include::partial$common-file-lock.adoc[]
 
+[NOTE]
+If the file is already locked by someone else, the connector cannot unlock it, resulting in a `FTP:FILE_LOCK` error.
```

Comment:
The connector can't unlock the file if it is locked by someone else, and you receive an `FTP:FILE_LOCK` error.